api/api.go

```diff
@@ -95,3 +100,13 @@ func GetPrompt(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func GetCard(w http.ResponseWriter, r *http.Request) {
+	cardName := chi.URLParam(r, "*")
```

Comment:
this is technically `promptName` - below we use the name of the file as `cardName`